mmv1/templates/terraform/examples/service_directory_namespace_ga.tf.tmpl

```diff
@@ -0,0 +1,9 @@
+resource "google_service_directory_namespace" "{{$.PrimaryResourceId}}" {
```

Comment:
I believe this is the same example as https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/templates/terraform/examples/service_directory_namespace_basic.tf.tmpl. There's no need to explicitly include an example for ga. It will be automatically generated for both ga and beta providers once the `min_version: 'beta'` flag is removed from the resource.